miden-tx/src/tests/kernel_tests/test_note.rs

```diff
@@ -520,3 +517,29 @@ fn test_get_inputs_hash() {
 
     assert_eq!(process.get_stack_state()[0..16], expected_stack);
 }
+
+#[test]
+fn test_get_current_script_hash() {
+    let tx_context = TransactionContextBuilder::with_standard_account(ONE)
+        .with_mock_notes_preserved()
+        .build();
+
+    // calling get_script_hash should return script hash
+    let code = "
+    use.kernel::prologue
+    use.miden::note
+
+    begin
+        exec.prologue::prepare_transaction
+        exec.note::get_script_hash
+
+        # truncate the stack
+        swapw dropw
+    end
+    ";
+
+    let process = tx_context.execute_code(code).unwrap();
+
+    let script_hash: Word = tx_context.input_notes().get_note(0).note().script().hash().into();
+    assert_eq!(process.stack.get_word(0), script_hash);
```

Comment:
I think we can avoid importing the `Word` here by calling the `.as_elements()` on the `RpoDigest` returned from the `.hash()`:
```Rust
let script_hash = tx_context.input_notes().get_note(0).note().script().hash();
assert_eq!(process.stack.get_word(0), script_hash.as_elements());
```